magenta-lib/src/main/scala/magenta/deployment_type/CloudFormation.scala

```diff
@@ -82,6 +89,15 @@ object CloudFormation extends DeploymentType {
       implicit val artifactClient = resources.artifactClient
       val reporter = resources.reporter
 
+      val amiParameterMap: Map[String, Map[String, String]] = (amiParametersToTags.get(pkg), amiTags.get(pkg)) match {
+        case (Some(parametersToTags), Some(tags)) =>
+          reporter.warning("Both amiParametersToTags and amiTags supplied. Ignoring amiTags.")
+          parametersToTags
+        case (Some(parametersToTags), _) => parametersToTags
+        case (None, Some(tags)) => Map(amiParameter(pkg, target, reporter) -> tags)
+        case _ => Map.empty
+      }
+
```

Comment:
Would be nice to avoid duplication, even though @DiegoVazquezNanini had explained to me this is quite hard to do in the current state.